src/test/java/com/deque/axe/ExampleTest.java

```diff
@@ -111,26 +111,6 @@ public void testAccessibilityWithOptions() {
 		}
```

Comment:
what is happening with this file here?

Reply:
After an hour of not getting the test to pass, I decided to remove it. Since we are deprecating this repo it should be fine.